### PR TITLE
8206 test coverage fix 

### PR DESCRIPTION
**Pull request by ale-cristofori**
Reintroduced strictly checking for undefined and
further check for timeline component visibility

## Description
Created this PR to merge changes into the `master` for https://github.com/geosolutions-it/mapstore2/issues/8206

### CODE REVIEW

**Comment by ale-cristofori** [comment]
As agreed, pass this to @alexander-fedorenko to complete additional test case implemented with latest commit

File: web/client/epics/__tests__/playback-test.js [hunk 1]
```diff
@@ -315,4 +315,24 @@ describe('playback Epics', () => {
...
+            try {
+                expect(actions.length).toBe(2);
+                expect(actions[0].type).toBe(TEST_TIMEOUT);
+                expect(actions[1].type).toBe('EPIC_COMPLETED');
+                done();
+            } catch (e) {
+                done(e);
+            }
+        }, state, false, true);
+    });
```
Comment (MV88) [inline web/client/epics/__tests__/playback-test.js:337]: can you add two tests 
when a CHANGE_LAYER_PROPERTIES happens on other layers with others properties different from visibility AND
- timeline is visible 
- timeline is not visible

**Review by MV88 — CHANGES_REQUESTED** [review]

File: web/client/epics/__tests__/playback-test.js (hunk 1 above)
Comment (alexander-fedorenko) [reply web/client/epics/__tests__/playback-test.js:337]: Thanks @MV88, added coverage for these cases

**Review by MV88 — APPROVED** [review]

**Comment by MV88** [comment]
@ElenaGallo please test in DEV

**Comment by ElenaGallo** [comment]
Test passed, @ale-cristofori please backport to 2022.01.xx.

**Pull request merged**